### PR TITLE
Update announcing releases available

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

**Comment by mhdawson** [comment]
@nodejs/website would like to send this out in next 1/2 hour.

File: locale/en/blog/vulnerability/december-2017-security-releases.md [hunk 1]
```diff
@@ -24,7 +75,7 @@ All versions of 4.x, 6.x, 8.x and 9.x are vulnerable to an issue to be fixed in
 # Release timing
```
Comment (refack) [inline locale/en/blog/vulnerability/december-2017-security-releases.md]: Should this be ## too?

**Review by MylesBorins — APPROVED** [review]
LGTM

File: locale/en/blog/vulnerability/december-2017-security-releases.md [hunk 2]
```diff
@@ -1,20 +1,71 @@
...
+
+**Note**: The build for ARMv6 for v8.9.3 is not yet available and will be released later today.
+
+## Data Confidentiality/Integrity Vulnerability - CVE-2017-15896
+
+Node.js was affected by OpenSSL vulnerability CVE-2017-3737 in regards to the use of SSL_read() due to TLS handshake failure. The result was that an active network attacker could send application data to Node.js using the TLS or HTTP2 modules in a way that bypassed TLS authentication and encryption.  
+
+* The original HTTP module was not affected.
+
+* The vulnerability in the HTTP2 module (which only existing in the 8.X and 9.X lines) was fixed through nodejs/node@f3686f2. HTTP2 was previously exploitable through the submission of malicious data by an attacker.
```
Comment (refack) [inline locale/en/blog/vulnerability/december-2017-security-releases.md]: Do you want to keep the revision as `nodejs/node@f3686f2` (or change to link to GitHub?)

**Review by refack — APPROVED** [review]

**Review by chris--young — APPROVED** [review]
Would be nice if the urls to openssl.org were anchors. Also maybe the sha for the commit that fixed the http2 bug.

File: locale/en/blog/vulnerability/december-2017-security-releases.md (hunk 1 above)
Comment (mhdawson) [reply locale/en/blog/vulnerability/december-2017-security-releases.md]: yes, will fix.

File: locale/en/blog/vulnerability/december-2017-security-releases.md [hunk 3]
```diff
@@ -1,20 +1,71 @@
...
 layout: blog-post.hbs
 author: Michael Dawson
 ---
 
-# Summary
+# _(Update 7-December-2017)_ Security releases available
+
+## Summary
+
+Updates are now available for all active Node.js release lines.  These include the fix for the vulnerability identified in the initial announcement.
```
Comment (daxlab) [inline locale/en/blog/vulnerability/december-2017-security-releases.md]: Extra whitespace `lines.   These` --> `lines. These `

**Comment by mhdawson** [comment]
pushed commits to address all comments so far

**Review by daxlab — APPROVED** [review]

**Review by bnb — APPROVED** [review]

**Comment by mhdawson** [comment]
Ok landing.

**Comment by mhdawson** [comment]
Landed as 31e4313f78b3cfc613e8b792bb764afc53b53945

**Pull request closed without merging**